.gitignore

```diff
@@ -35,6 +35,7 @@ test/**/next-env.d.ts
 test/tmp/**
 test/.trace
 test/traces
+test/development/slow-module-detection/pages/slowModule.js
```

Comment:
you can add a gitignore in that test folder, I think it will be easier to manage tests when we move them around